lib/matplotlib/backends/qt_compat.py

```diff
@@ -131,9 +140,8 @@ def _getSaveFileName(*args, **kwargs):
             def _getSaveFileName(*args, **kwargs):
                 return QtGui.QFileDialog.getSaveFileName(*args, **kwargs), None
 
-    else:  # PyQt5 API
-        from PyQt5 import QtCore, QtGui, QtWidgets
-        _getSaveFileName = QtWidgets.QFileDialog.getSaveFileName
+    else:
+        raise RuntimeError("No PyQt bindings found despite sip importing")
```

Comment:
Could we add some more details here. I guess the solution is to either install pyqt4/5, uninstall sip or explicitly set the QT_API env variable to QT_API_PYSIDE